pkg/skaffold/schema/v3alpha1/config.go

```diff
@@ -0,0 +1,1405 @@
+/*
```

Comment:
If we're not going to immediately release this config I think we should move it to `pkg/skaffold/schema/experimental` or something. I think that'll help running into issues with all the `make` targets that do releases, new configs, etc.

Reply:
This new version will go public in the next skaffold release (May 13th). It will allow early v3 users to try it out and won't mess up the current user experience.

Here's a corresponding PR. https://github.com/GoogleContainerTools/skaffold/pull/5694
Basically, v3 users run `skaffold dev --experimental`, v2 users run `skaffold dev`. 
New v3 release will bump the v3 master version v3alpha1 v3alpha2, while we also bump v2 versions for each month release v2beta14, v2beta15. v2 and v3 are **incompatible** with each other.

Thank you for raising this concern @gsquared94. I'll double check the make rules. BTW, the make rules are pretty cool.(my CLs have broken quicktest, coverage, linters, so far. Really nice coverage! 😄)  
 
cc @viglesiasce @tejal29 @nkubala  @marlon-gamez @aaron-prindle

Reply:
Thanks @yuwenma for  bringing this up. 

We are now moving to 2 weeks release cycle and this might get published before May 13th.
That said, we are running behind on V3 release plan. 
I don't mind the version number v2alpha1, v3alpha2 as alpha signifies experimental phase and users can choose if they want to move to the next experimental config or else config management would be confusing for users.

Like @marlon-gamez pointed previously, we should not point `v3alpha*` as latest. Can you address that in a separate PR?
Since this is the first version, we could do any from below
1) Make the `v3` flag hidden and not publish any docs for `v3alpha*` config,.
2) Make the `v3` flag hidden, create a directory https://skaffold-latest.firebaseapp.com/docs/references/yaml/v3/

When we do announce skaffold kpt integration, May 13th potentially
1) Unhide `v3` flag and publish  first `v3alpha1` schema version in https://skaffold-latest.firebaseapp.com/docs/references/yaml/v3/
2) From there on, any config changes would incremental config version

Reply:
Freezing the v3 config after the announcement of kpt skaffold integration would help you reiterate on the v3 config until the release without worrying about versioning.

Reply:
@tejal29 

a qq: "we should not point v3alpha* as latest. Can you address that in a separate PR?" The `latest/config.go` is pointed to v2beta*. Do you mean somewhere else?

FYI, there's [doc](https://docs.google.com/document/d/1GFy_5ya-ajjSVe1t6f7IWUlAQu35eTHAZZ1GkJBiqcA/edit?resourcekey=0-sKE-yXjuymQ0w2qqNXnTzw) talking more about the refactoring. Let me know if that makes sense and I can update this PR accordingly.

Reply:
reviewed the doc. This can now be resolved.